Source/Core/Gen4/Encounters4.cpp

```diff
@@ -31,6 +30,27 @@
 
 namespace Encounters4
 {
+    bool getHeadbuttSpecialFlag(Game game, int location)
```

Comment:
Please move this function down below getEncounters()

Reply:
> Please move this function down below getEncounters()

done